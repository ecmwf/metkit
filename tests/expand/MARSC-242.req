retrieve,
    DOMAIN=G,LEVTYPE=ML,LEVELIST=1,DATE=JAN,PARAM=155.128,CLASS=E4,TYPE=CL,STREAM=MODA,EXPVER=0001,
    target=reference.data 