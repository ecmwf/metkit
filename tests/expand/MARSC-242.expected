RETRIEVE,
    CLASS      = E4,
    TYPE       = CL,
    STREAM     = MODA,
    EXPVER     = 0001,
    REPRES     = SH,
    LEVTYPE    = ML,
    LEVELIST   = 1,
    PARAM      = 155,
    DATE       = JAN,
    DOMAIN     = G,
    TARGET     = reference.data